ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/xpn/byo-hosted-zone/private/deprovision/cucushift-installer-rehearse-gcp-ipi-xpn-byo-hosted-zone-private-deprovision-chain.yaml

```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-xpn-byo-hosted-zone-private-deprovision
```

Comment:
This is used to replace `cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-hosted-zone-deprovision`, right ? if yes, I guess we need remove `cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-hosted-zone-deprovision`,